localstack/services/awslambda/lambda_executors.py

```diff
@@ -147,6 +149,10 @@ def run_lambda_executor(self, cmd, event=None, env_vars={}):
 
         return result, log_output
 
+    def port(self):
```

Comment:
I think we should remove that method, looks like it is not being used. (also, it potentially aliases the variable `self.port` with the same name)